kythe/rust/indexer/tests/providers.rs

```diff
@@ -50,7 +50,9 @@ fn test_kzip_provider() {
     // Check the `contents` function
     let contents_result = kzip_provider.contents(file_hash);
     assert!(!contents_result.is_err());
-    assert_eq!(contents_result.unwrap(), "Test\n", "File contents did not match expected contents");
+    let contents_string =
+        String::from_utf8(contents_result.unwrap()).expect("File contents was not valid UTF-8");
+    assert_eq!(contents_string, "Test\n", "File contents did not match expected contents");
```

Comment:
This assert is actually better than the one on line 58 below.  If that one fails it won't tell you what the contents are.  This one, though, will tell you exactly what is wrong.

Reply:
One thing I can do for the assertion on line 58 is to try including the contents in the assertion error message. The issue is that I would need to unwrap the binary contents (which shouldn't cause a panic because we would assume the result is Ok) and attempt to parse a UTF-8 string from it, which might fail and cause a panic. One mitigation might be to just print out the Vec<u8> contents instead of trying to parse as a string. However, another option would be to turn that part of the test into more complex if/else logic and then just `panic!` with a specific error message depending on the resulting logical branch. Do any of these seem like good ideas?

Reply:
Printing vec contents is quite enough if possible.  There's no need to be super-elaborate: simply revealing the information one needs for debugging would be enough.

Reply:
Okay, the test will now print out the vector contents in the emitted error message